src/test/java/edu/cornell/kfs/concur/batch/xmlObjects/TestPdpMarshal.java

```diff
@@ -0,0 +1,69 @@
+package edu.cornell.kfs.concur.batch.xmlObjects;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.IOException;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.StringUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.xml.sax.SAXException;
+
+import edu.cornell.kfs.concur.batch.fixture.PdpFeedFileBaseEntryFixture;
+import junit.framework.Assert;
+
+public class TestPdpMarshal {
+
+    private static final String BATCH_DIRECTORY = "test/opt/work/staging/concur/standardAccountingExtract/pdpOutput/";
+    private static final String EXAMPLE_PDP_FILE_PATH = "src/test/java/edu/cornell/kfs/concur/batch/fixture/PdpExample.xml";
+
+    private File batchDirectoryFile;
+
+    @Before
+    public void setUp() throws Exception {
+        batchDirectoryFile = new File(BATCH_DIRECTORY);
+        batchDirectoryFile.mkdir();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        FileUtils.deleteDirectory(batchDirectoryFile);
+    }
+
+    @Test
+    public void test() throws JAXBException, IOException, SAXException, ParserConfigurationException {
+        PdpFeedFileBaseEntry pdpFile = PdpFeedFileBaseEntryFixture.buildPdpFile();
```

Comment:
I think both batch jobs will need to call code similar to this to generate the xml file. Should this be extracted in a service that could be reused? Thank you.